addons/l10n_ch/models/res_bank.py

```diff
@@ -286,6 +290,16 @@ def _is_qr_reference(self, reference):
                and re.match('\d+$', reference) \
                and reference == mod10r(reference[:-1])
 
+    @api.model
+    def _is_iso11649_reference(self, reference):
+        """ Checks whether the given reference is a ISO11649 (SCOR) reference.
+        """
+        return reference \
+               and len(reference) >= 5 \
+               and len(reference) <= 25 \
+               and reference.startswith('RF') \
+               and stdnum.iso7064.mod_97_10.validate(reference[4:] + reference[:4])
```

Comment:
My version of stdnum (the one from the requirements) doesn't contain iso7064 and crashes here.

Reply:
removed dependency on iso7064